src/Resources/translations/flashes.en.yml

```diff
@@ -0,0 +1,3 @@
+sylius:
+    reorder:
+        items_price_changed: 'Price of some order items has changed. It may have affected order total. Previous order total: %order_total%'
```

Comment:
1. `Price of (...) order items` is not exactly true, what have changed are prices of product variants
2. `It may have affected order total` I wonder is there any situation that changing prices of variants does not affect the order total 😄 Maybe something like `Prices of some products has changed, which have affected order total`?